keyboard/src/data/keyboard_raw.ycp

```diff
@@ -527,8 +527,7 @@ return ($[
 	// keyboard layout
 	_("Khmer"),
 	$[
-	    // This used to be khmer.map.gz, but that is only a link to us.map.gz
```

Comment:
Instead of deleting the comment, I would turn it into a warning to prevent us from making the same error again. Something like:

```
// Although khmer.map.gz is only a link to us.map.gz, it's important to keep the
// khmer name because is used by yast2-x11 to to obtain the proper XkbLayout
```

Reply:
Makes sense, thank you!